### PR TITLE
feat!: combine all helm charts

### PR DESCRIPTION
**Pull request by olevski**
/deploy #persist renku=feat-combine-all-helm-charts extra-values=initDb.image.repository=registry.dev.renku.ch/tasko.olevski/renku-images/init-db,initDb.image.tag=0.36.4-8.21cbfb5,keycloakx.initRealm.image.repository=registry.dev.renku.ch/tasko.olevski/renku-images/init-realm,keycloakx.initRealm.image.tag=0.36.4-8.21cbfb5

### CODE REVIEW

File: .github/workflows/acceptance-tests.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
       name: renku-ci-gr-${{ github.event.number }}
     steps:
       - name: deploy-pr
-        uses: SwissDataScienceCenter/renku-actions/deploy-renku@v1.6.0
+        uses: SwissDataScienceCenter/renku-actions/deploy-renku@feat-combined-helm-charts
```
Comment (jachro) [inline .github/workflows/acceptance-tests.yaml]: I believe the `feat-combined-helm-charts` is something temporal and will be updated with a new release of renku-actions. Is that the intention?

**Comment by jachro** [comment]
@olevski I'm not sure how you want to proceed with merging this PR. We are about (today or tomorrow morning) to release a bugfix release and my question is whether we should merge this PR now or wait a bit. I mean, anyway, you've got a green light from us but our only concern is about the timing.

File: .github/workflows/acceptance-tests.yaml (hunk 1 above)
Comment (olevski) [reply .github/workflows/acceptance-tests.yaml]: Now it is 1.7.0

**Comment by olevski** [comment]
@jachro I am planning to go over the changes, do review and merge stuff during a breakout session tomorrow in Bern. But dont worry you can release as much or as little as you want in the meantime. Even if I do not manage to merge everything tomorrow.

**Review by jachro — APPROVED** [review]

**Comment by olevski** [comment]
The failing sessions test is because of a bug we introduced in the notebooks with the hibernating sessions features. It is being fixed.

**Pull request merged**